### PR TITLE
addon:vela-prism

### PR DESCRIPTION
**Pull request by oeular**
Signed-off-by: Xueliang Yang <yangxueliang@cmbchina.com>

### Checklist

I have:

- [ ] Title of the PR starts with type (e.g. `[Addon]` or `[Trait]`).
- [ ] Updated/Added any relevant [documentation] and [examples].
- [ ] Unit/E2E Tests added.

### CODE REVIEW

**Comment by oeular** [comment]
ping @Somefive Can you have a review ?

**Comment by fourierr** [comment]
what about the helm parameter  be exposed ? So user can modify it by addon.

**Comment by oeular** [comment]
Maybe we can control the replicas and secureTLS. @fourierr

File: addons/trait-pyroscope/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Supported workload type
+Pyroscope Trait supports following component types: webservice、worker and cloneset.
+
+# How to start
+- Use a component typed webservice to start, keep the following to app-demo.yaml, then vela up -f app-demo.yaml
```
Comment (wangyikewxgm) [inline addons/trait-pyroscope/README.md:5]: why we need these trait-pryoscope files in this addon?

Comment (oeular) [reply addons/trait-pyroscope/README.md:5]: I will close this pr, new a pr instead: https://github.com/kubevela/catalog/pull/364#issue-1261527441

**Comment by Somefive** [comment]
Sorry for the late response. It is amazing to create the vela-prism addon. Generally looks good to me. Several advices:
1. As @fourierr said, it would be better if more helm values are exposed. I think `image.*` and `resource.*` can be included in the input properties of the addon.
2. Another thing to mention is that I was thinking if there is anyway possible to workaround the helm. Since helm requires fluxcd to be installed first. However, since there is a `pre-hook` and `post-hook` for the installation of vela-prism, so it might need to leverage workflow if we do not use helm component, which as far as I see, has not been supported in vela addon mechanism yet.

**Pull request closed without merging**